pdp/static/js/pdp_controls.js

```diff
@@ -179,6 +180,35 @@ Colorbar.prototype = {
         }
     },
 
+    round_to_nearest: function (mult) {
+      this.minimum = Math.round(this.minimum / mult) * mult;
```

Comment:
Yes, but functions are meant to be called multiple times :) That's part of what tips you off to the fact that something should be a function... it's reused over and over.